ins-node/test/ins_mock/ESP8266WiFi.h

```diff
@@ -0,0 +1,127 @@
+#ifndef ESP8266WIFI_H
+#define ESP8266WIFI_H
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include <string>
+
+// Some stubbed variables and types
+const uint8_t WIFI_STA = 0;
+const uint8_t WL_CONNECT_FAILED = 0;
+const uint8_t WL_CONNECTED = 1;
+using String = std::string; // Create an alias for Arduino String
```

Comment:
Using `using` here in this header file does not have any unwanted side-effects since this is only used in the tests as a replacement for Arduino's String class.